### ISSUE TITLE
Bubble sort in Java

### ISSUE THREAD

**Issue by GayanSampathManamendra** [FOSS-UCSC/FOSSALGO#3]
This issues is about bubble sort in java.https://www.geeksforgeeks.org/bubble-sort/

- [x]  Issue title form Algo/DS Name [Language]

- [x] I searched or browsed the repo's other issues to ensure this is not a duplicate

- [x]  This Algo/DS is worth to add this repository

- [x]  Assign this issue to me

- [x]  I will send my PR only after I'm assigned to this issue by any maintainer

**Comment by RavinduSachintha** [comment]
@GayanSampathManamendra  , your contribution was hide due to the renaming of folder. Please if you can make again another pull request for this issue.. Change the file name exactly as "bubbleSort.java"
